src/pages/about/aboutView.js

```diff
@@ -1,24 +1,24 @@
 import React from 'react';
-import { AboutHeader, AboutBody } from 'bento-components';
+import { AboutBody } from 'bento-components';
 import { withStyles } from '@material-ui/core';
-import Stats from '../../components/Stats/AllStatsController';
+import AboutHeader from './aboutHeader';
```

Comment:
@jonkiky Should we import this new custom header to bento-tools ?

Reply:
I thought about this. But this Header does not have any functions, only the ICDC specific style. 
Less reusable in other data commons.  So put it as optional.